Makefile

```diff
@@ -33,6 +33,10 @@ GO_BUILD_LDFLAGS = -X github.com/apache/skywalking-$(PROJECT)/commands.version=$
 PLATFORMS := windows linux darwin
 os = $(word 1, $@)
 
+artifact_linux := $(PROJECT)
+artifact_darwin := $(PROJECT)
+artifact_windows := $(PROJECT).exe
```

Comment:
😉 https://github.com/apache/skywalking-eyes/pull/101/files

Reply:
Ha, much more elegant, I'm gonna change to this, don't merge.